src/main/java/teammates/common/datatransfer/attributes/FeedbackResponseAttributes.java

```diff
@@ -58,7 +58,7 @@ public FeedbackResponseAttributes(String feedbackSessionName,
         this.giverSection = giverSection;
         this.recipient = recipient;
         this.recipientSection = recipientSection;
-        this.responseMetaData = responseMetaData;
+        this.responseDetails = deserializeResponseFromMetaData(responseMetaData);
```

Comment:
You can pass `FeedbackResponseDetail responseDetail` as parameter for the method?

Reply:
This constructor is only used in Testing, and is used to construct a custom FeedbackResponseAttribute conveniently

Reply:
Why should we introduce the exception in production code for testing purpose?

Reply:
Then the alternative for existing tests is to create a `FeedbackResponse` entity and pass it to create `FeedbackResponseAttribute` object. Is this acceptable?

Reply:
For attribute test?

Reply:
I don't see any problem here.

For example, in this case
```
        FeedbackResponseAttributes responseToAdd =
                new FeedbackResponseAttributes("First feedback session", "idOfTypicalCourse1",
                                               getQuestionFromDatastore("qn1InSession1InCourse1").getId(),
                                               FeedbackQuestionType.TEXT, studentToUpdate.email, "Section 1",
                                               studentToUpdate.email, "Section 1", "New Response to self");
```

You can just create a new FeedbackResponseTextDetail and pass it to the constructor (replace `"New Response to self"`).

Reply:
Ok I see. I misunderstood your thought just now